db/migrate/20200116162334_create_line_items.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateLineItems < ActiveRecord::Migration[6.0]
+  def change
+    create_table :line_items do |t|
+      t.integer :quantity, default: 1
+      t.integer :book_id
```

Comment:
We could define relations in such way when they are optional. Otherwise, it should be a foreign key

Reply:
done